kubetest2-tf/data/k8s-ansible/roles/install-k8s/templates/kubeadm.yaml.j2

```diff
@@ -17,12 +17,17 @@ nodeRegistration:
   criSocket: "unix:///run/crio/crio.sock"
 {% endif %}
   kubeletExtraArgs:
-    cgroup-driver: {{ cgroup_driver }}
+    - name: cgroup-driver
+      value: {{ cgroup_driver }}
 {% if (feature_gates is defined) %}
-    feature-gates: {{ feature_gates }}
+    - name: feature-gates
+      value: "{{ feature_gates | join(',') }}"
```

Comment:
The corresponding entry in group_vars/all would be feature_gates=["featureA=true","featureB=true"]

This is done to allow iterating through the feature gates for listing under kubelet's extraArgs- feature flags: Ref from [kubespray](https://github.com/kubernetes-sigs/kubespray/blob/79fbfdf271ea3494daab0adb00b5c4cb61353763/roles/kubernetes/control-plane/templates/kubeadm-config.v1beta4.yaml.j2#L588-L593)